saturnbase-rstudio-workbench-gpu-11.1/install-miniconda.bash

```diff
@@ -4,7 +4,7 @@ set -ex
 
 cd $(dirname $0)
 
-MINICONDA_VERSION=py37_4.9.2
+MINICONDA_VERSION=py38_4.9.2
```

Comment:
Any reason to go to 3.8 instead of 3.9 or 3.10?

Reply:
no strong reason for me. I can check that there are builds for everything in python 3.9